gapic/src/main/com/google/gapid/perfetto/views/TrackContainer.java

```diff
@@ -208,7 +213,7 @@ public void stop() {
   }
 
   private static class Group<T extends CopyablePanel<T> & TitledPanel, D extends CopyablePanel<D>>
-      extends Panel.Base implements CopyablePanel<Group<T, D>> {
+      extends Panel.Base implements CopyablePanel<Group<T, D>>, Panel.Grouper, FilterablePanel {
     private final T summary;
     private final CopyablePanel.Group detail;
     protected final BiConsumer<CopyablePanel.Group, Boolean> filter;
```

Comment:
Unable to highlight more unchanged lines here. But can we rename the members filter and filtered in this class since their purpose is for expand/collapse? It's a little confusing as we are using the word filter differently now.

Reply:
Done.